eng/scripts/Language-Settings.ps1

```diff
@@ -8,20 +8,16 @@ $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=con
 
 . "$PSScriptRoot/docs/Docs-ToC.ps1"
 
-function Confirm-NodeInstallation
```

Comment:
nit: any change in this file, or just formatting? If latter maybe exclude it from this PR.

Reply:
There is a change to remove the exclusion of the asynciterator polyfill.